CODEOWNERS

```
@@ -66,6 +66,9 @@
 /modules/solid_properties @joshuahansel
 /modules/stochastic_tools @zachmprince
 /modules/tensor_mechanics @bwspenc @dschwen @jiangwen84
+/modules/tensor_mechanics/*/lagrangian @hugary1995
+/modules/tensor_mechanics/*/materials/cohesive_zone_model @hugary1995
+/modules/tensor_mechanics/*/interfacekernels @hugary1995
```

```suggestion
/modules/tensor_mechanics/*/lagrangian @bwspenc @dschwen @jiangwen84 @hugary1995
/modules/tensor_mechanics/*/materials/cohesive_zone_model @bwspenc @dschwen @jiangwen84 @hugary1995
/modules/tensor_mechanics/*/interfacekernels @bwspenc @dschwen @jiangwen84 @hugary1995
```

CODEOWNERS only uses one line per file, the more selective one. So.. in order to get

```
/modules/tensor_mechanics @bwspenc @dschwen @jiangwen84
```

in your entries, you need to also add it